### ISSUE TITLE
Selecting a country that contains spaces fails

### ISSUE THREAD

**Issue by wradcliffe** [jsbain/worldmap#1]
Selecting any country that has spaces in the name fails to return anything to the webview delegate. Most likely due to there being no code to uuencode the world: url.

**Comment by jsbain** [comment]
I cannot reproduce this.  Which country did you select?  What ios version, pythonista version are you using?

```
country selected: United States
country selected: Russian Federation
country selected: Australia
country selected: New Zealand
country selected: South Africa
country selected: Saudi Arabia
```

**Comment by wradcliffe** [comment]
Pythonista 1.6 - IOS 8.1.3 - "United States"

Must be something with 1.6

**Comment by jsbain** [comment]
Does it work on other countries?  I wonder if there is some kind of ios 8 security changes, or javascript changes. We could try creating more of a well formed uri, like

`'world://show?argv=' + countryname`

**Comment by wradcliffe** [comment]
Yes - it works fine for any country that has no spaces in the name.  That's
why I thought it was a URL encode problem. I was surprised that it works
fine for you, but you are still on 1.5 - right?

**Comment by wradcliffe** [comment]
Here is a fix to index.html that encodes the URL and turns the space to a
%20:

window.location.assign('world://'+encodeURIComponent(worldmap.names[this.attr("title")]));
